lua/autorun/server/cfc_attention_monitoring_init.lua

```diff
@@ -0,0 +1,27 @@
+local tabbedOutPlys = {}
+local hasDataChanged = false
+
+util.AddNetworkString( "CFC_AttentionMonitor_GameHasFocus" )
+util.AddNetworkString( "CFC_AttentionMonitor_SendData" )
+
+hook.Add( "PlayerDisconnected", "CFC_AttentionMonitor_PlayerLeave", function(ply)
```

Comment:
This is nitpicking but I thought i would mention it. If you are naming hooks, names like this dont provide any extra information. in `CFC_AttentionMonitor_PlayerLeave` PlayerLeave is basically saying  the same thing as PlayerDisconected. 
Instead you could name it something like `CFC_AttentionMonitor_CleanupPlayerData` this tells you what this callback actually does, you dont need to call it  PlayerLeave because its already a PlayerDisconected hook.

tldr
CleanupPlayerData, DeletePlayerData, etc is better than PlayerLeave because it tells you what the function actually does and does not repeat information

Reply:
okie